maps/sccv_horizon/code/sccv_horizon_overmap.dm

```diff
@@ -175,3 +194,42 @@
 	icon_keyboard = null
 	circuit = null
 
+//
+// ------------------------------------------- quark
+//
+
+/obj/effect/overmap/visitable/ship/landable/quark
+	name = "Quark"
+	class = "SCCV"
+	designation = "Quark"
+	desc = "..."
+	shuttle = "Quark"
+	icon_state = "pod" // <-------------- temporary, hopefully someone sprites it some day
+	moving_state = "pod_moving"
+	colors = list("#cfd4ff", "#78adf8")
+	// scanimage = "quark.png" // <-------------- hopefully someone sprites it some day
+	designer = "Hephaestus Industries, NanoTrasen"
+	volume = "14 meters length, 7 meters beam/width, 5 meters vertical height"
+	weapons = "Extruding fore-mounted low-caliber ballistic rotary armament"
+	sizeclass = "Jester-type Scout Skiff"
+	shiptype = "Exploratory survey and scouting, high-speed target interception"
+	max_speed = 1/(3 SECONDS)
+	burn_delay = 2 SECONDS
+	vessel_mass = 3000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_TINY
+
+// /obj/effect/overmap/visitable/ship/landable/quark/get_skybox_representation()
+// 	var/image/skybox_image = image('icons/skybox/subcapital_ships.dmi', "quark")
+// 	skybox_image.pixel_x = rand(0,64)
+// 	skybox_image.pixel_y = rand(128,256) // <---------- hopefully someone sprites it some day
+// 	return skybox_image
+
+/obj/machinery/computer/shuttle_control/explore/terminal/quark
+	name = "\improper Quark control console"
+	shuttle_tag = "Quark"
+	req_access = list(ACCESS_XENOARCH, ACCESS_RESEARCH)
+
+//
+// ------------------------------------------- fin
+//
```

Comment:
Pointless section, we can see it's the end of the file

Reply:
awgjtriashjio-0trasemkjhpiea4kt 392 JTO[0I34AJ gy-90h3jw=09HJA3E049I YJHA-0I9EKJ YH-=0923wt j90snj 9054ej=0 y9hj a3e-=0 9jae50-9gjh a=9e 

fine